render/egl.c

```diff
@@ -272,6 +272,14 @@ struct wlr_egl *wlr_egl_create(EGLenum platform, void *remote_display,
 			goto error;
 		}
 
+		if (check_egl_ext(device_exts_str, "EGL_MESA_device_software") &&
+				!getenv("WLR_RENDERER_ALLOW_SOFTWARE")) {
```

Comment:
Can we only lift the check if the env var is exactly "1"? Other `WLR_` env vars ignore other values (like "0").

Reply:
Sure!